content/Assets/Styles/components/card/_default.scss

```diff
@@ -3,11 +3,37 @@
    ========================================================================== */
 
 .card {
+    border: 1px solid var(--colorGrey);
+    display: block;
     margin-left: auto;
     margin-right: auto;
     position: relative;
 
+    background-color: var(--colorWhite);
+
+    color: var(--colorTextDefault);
+    text-decoration: none;
+
+    &__media {
+        &-image {
+            min-width: 100%;
+        }
+    }
+
     &__body {
-        padding-top: $content-gutter-small;
+        padding: $card-padding;
+    }
+
+    &__title-area {
```

Comment:
@Newnab What do you think of this being `card__header` instead?

Reply:
I guess at the moment `card__body` is more consistent with `hero__body` and `media__body`. This would be the only place we say `__header`.

Reply:
If you mean swapping `title-area` for `header` I have no strong feelings either way... I would typically avoid header because that's the name of a tag (and not the tag that this currently uses, although maybe you want to change that too?). Happy to swap if others feel header would be better.

Reply:
I'd be happy with `<header class="card__header">...</header>`.

Reply:
My bad, I misunderstood 🤦🏻‍♂️

`<header class="card__header">...</header>` sounds good to me. I like the use of `header`, too.